library/junos_install_config

```diff
@@ -250,6 +261,9 @@ def _load_via_netconf(module):
         logging.error(msg)
         module.fail_json(msg=msg)
 
+    if args['ignore_warning'] in BOOLEANS + ["True", "False"]:
+        args['ignore_warning'] = module.boolean(module.params['ignore_warning'])
+
```

Comment:
I didnt do that for few reason:
older version of ansible will have it as list itself. with >=2.3 it will be string and in PyEZ we do regular expression check where we loop with all rpc-warning and check if that matches any on in the list.
In this case it will just search/match the warnings in the string of list (for ex "['statment not found', 'xyz']"